workflows/proteomics/metaproteomics-gtn/metaproteomics_gtn_wf.ga

```diff
@@ -0,0 +1,724 @@
+            "label": "Search GUI",
+            "name": "Search GUI",
+            "outputs": [
+                {
+                    "name": "searchgui_results",
+                    "type": "searchgui_archive"
+                }
+            ],
+            "position": {
+                "left": 228.84373957916367,
+                "top": 14.484374745319764
+            },
+            "post_job_actions": {},
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/galaxyp/peptideshaker/search_gui/3.3.10.1",
+            "tool_shed_repository": {
+                "changeset_revision": "7fdd9119cc4f",
+                "name": "peptideshaker",
+                "owner": "galaxyp",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"advanced_options\": {\"searchgui_advanced\": {\"searchgui_advanced_selector\": \"basic\", \"__current_case__\": 0}, \"xtandem\": {\"xtandem_advanced\": \"yes\", \"__current_case__\": 1, \"xtandem_npeaks\": \"50\", \"xtandem_min_peaks\": \"15\", \"xtandem_min_frag_mz\": \"200\", \"xtandem_min_prec_mass\": \"200\", \"xtandem_noise_suppr\": true, \"xtandem_dynamic_range\": \"100\", \"xtandem_quick_acetyl\": false, \"xtandem_quick_pyro\": false, \"xtandem_stp_bias\": false, \"xtandem_evalue\": \"100.0\", \"xtandem_output_proteins\": false, \"xtandem_output_sequences\": false, \"xtandem_output_spectra\": true, \"xtandem_refine\": {\"xtandem_refine_selector\": \"no\", \"__current_case__\": 0}}, \"omssa\": {\"omssa_advanced\": \"no\", \"__current_case__\": 0}, \"msgf\": {\"msgf_advanced\": \"no\", \"__current_case__\": 0}, \"ms_amanda\": {\"ms_amanda_advanced\": \"no\", \"__current_case__\": 0}, \"tide\": {\"tide_advanced\": \"no\", \"__current_case__\": 0}, \"myrimatch\": {\"myrimatch_advanced\": \"no\", \"__current_case__\": 0}, \"comet\": {\"comet_advanced\": \"no\", \"__current_case__\": 0}, \"directtag\": {\"directtag_advanced\": \"no\", \"__current_case__\": 0}, \"novor\": {\"novor_advanced\": \"no\", \"__current_case__\": 0}}, \"input_database\": {\"__class__\": \"ConnectedValue\"}, \"peak_lists\": {\"__class__\": \"ConnectedValue\"}, \"precursor_options\": {\"precursor_ion_tol_units\": \"1\", \"precursor_ion_tol\": \"10.0\", \"fragment_tol_units\": \"0\", \"fragment_tol\": \"0.2\", \"min_charge\": \"2\", \"max_charge\": \"6\", \"forward_ion\": \"b\", \"reverse_ion\": \"y\", \"min_isotope\": \"0\", \"max_isotope\": \"1\"}, \"protein_database_options\": {\"create_decoy\": true, \"use_gene_mapping\": false, \"update_gene_mapping\": false}, \"protein_digest_options\": {\"digestion\": {\"cleavage\": \"0\", \"__current_case__\": 0, \"digests\": [{\"__index__\": 0, \"enzyme\": \"Trypsin\", \"missed_cleavages\": \"2\"}]}}, \"protein_modification_options\": {\"fixed_modifications\": [\"Carbamidomethylation of C\"], \"variable_modifications\": [\"Oxidation of M\"]}, \"search_engines_options\": {\"engines\": [\"X!Tandem\"]}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
```

Comment:
"enzyme" and (probably) "missed_cleavages" should be WF input parameters for more general usability I guess.
No idea about the protein_modification_options and the search_engine, but maybe these too?

Reply:
Protein modification options are workflow parameters however Searchgui has options of using multiple search engines so that's a choice that the user has to make

Reply:
made changes to workflow